orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/HttpClientUtils.java

```diff
@@ -89,6 +92,13 @@ public long getRetryInterval() {
       return executionCount <= MAX_RETRIES;
     });
 
+    String proxyHostname = System.getProperty(JVM_HTTP_PROXY_HOST);
+    if (proxyHostname != null) {
+      int proxyPort = System.getProperty(JVM_HTTP_PROXY_PORT) != null ? Integer.parseInt(System.getProperty(JVM_HTTP_PROXY_PORT)) : 8080;
+      LOGGER.info("Found system properties for proxy configuration. Setting up http client to use proxy with hostname {} and port {}", proxyHostname, proxyPort );
```

Comment:
Can you break this log line up at 120 char? Aside from that, this looks great, thanks!

Reply:
Done! Thanks